src/fetch/period_slippage.py

```diff
@@ -103,8 +123,24 @@ def __init__(self) -> None:
     def from_data_set(cls, data_set: list[dict[str, str]]) -> SplitSlippages:
         """Constructs an object based on provided dataset"""
         results = cls()
-        for row in data_set:
-            results.append(slippage=SolverSlippage.from_dict(row))
+        # Intercept the constructor to merge results.
+        all_slippages = [SolverSlippage.from_dict(row) for row in data_set]
+        indexed_slippage = index_by(all_slippages, "solver_address")
+        for name, (old, new) in MERGE_DATA.items():
+            # Remove the old one.
+            old_address, target = Address(old), Address(new)
+            old_solver_slippage: SolverSlippage = indexed_slippage.pop(
```

Comment:
I think this is good enough for today. But for a future version, we could ensure that there is really only one element.

Reply:
I am not sure what you mean here...? there will only ever be one element (at most) in a mapping.